compact_proof/seal_to_json/Cargo.toml

```diff
@@ -26,3 +24,6 @@ tracing = "0.1"
 [features]
 cuda = ["risc0-zkvm/cuda", "risc0-zkp/cuda"]
 metal = ["risc0-zkvm/metal", "risc0-zkp/metal"]
+
+[package.metadata.release]
+release = false
```

Comment:
We may wactually want to publish this crate so that Bonsai can directly use it

Reply:
I don't think we need a separate `seal_to_json` crate. I think we could embed it within `risc0-zkvm`.

Reply:
Or there's also an idea @nategraf had which was to put all the Groth16 stuff into `risc0-groth16`, which I think this could go into.